docs/index.rst

```diff
@@ -13,15 +13,34 @@ distribution functions. The tools are put together via a ``streamz`` pipeline.
 Installation
 ============
 
+``xpdtools`` is installable via the ``conda`` package manager.
+If you don't have Anaconda (or miniconda) installed please follow the instructions
+from `Data Carpentry <https://datacarpentry.org/2016-05-29-PyCon/install.html>`_.
+
+With Anaconda or miniconda please enter into a terminal (on Windows Anaconda
```

Comment:
hand-holding here for non computer savvy people, maybe put step by step instructions?

e.g., on windows:
 - in the windows search box type "terminal" and select "Command Prompt"

Reply:
Can we loop around to this?